tests/instances.spec.ts

```diff
@@ -188,31 +168,45 @@ test("instance edit cloud init configuration", async ({ page }) => {
   await assertCode(page, "Network config", "foo:");
   await assertCode(page, "User data", "bar:");
   await assertCode(page, "Vendor data", "baz:");
+});
 
+test("instance terminal operations", async () => {
+  const instance = randomInstanceName();
+  await createInstance(page, instance);
+  await visitAndStartInstance(page, instance);
+  await page.getByTestId("tab-link-Terminal").click();
+  await expect(page.locator(".xterm-screen")).toBeVisible(TIMEOUT);
+  await page.keyboard.type("lsb_release -a");
+  await page.keyboard.press("Enter");
+  await expect(page.locator(".xterm-rows")).toContainText("Ubuntu");
+  let dialogPresent = false;
+  page.on("dialog", (dialog) => {
+    expect(dialog.type()).toEqual("confirm");
+    dialogPresent = true;
+    void dialog.accept();
+  });
+  await page.getByTestId("tab-link-Overview").click();
+  expect(dialogPresent).toEqual(true);
   await deleteInstance(page, instance);
 });
 
-test("instance create vm", async ({ page }) => {
+const vmInstance = randomInstanceName();
```

Comment:
Same as above - also not sure why this one is out of the test function scope. It seems to me that it doesn't get deleted also.

Reply:
Sorry my bad, I was experimenting with this one and forgot to move the `vmInstance` creation into the `beforeAll` block. Made the change now.